### ISSUE TITLE
A reference to site URL in meta tag is incorrectly recognised as a static resource

### ISSUE THREAD

**Issue by n1313** [hollandben/grunt-cache-bust#174]
My site is branded after its own URL. Let's say I call it Awesome.Site (and it is on https://awesome.site) and I have the following meta tags:

`<meta property="og:title" content="Awesome.Site"/>
<meta property="og:description" content="Let Awesome.Site help you with your troubles"/>`

After upgrading to 0.6.1 I have the following errors in my logs when I try to cache bust my index.html (where `./dist/` is my basedir):

```
>> Static asset "dist/Awesome.Site" skipped because it wasn't found, original reference=Awesome.Site
>> Static asset "dist/Let Awesome.Site help you with your troubles" skipped because it wasn't found, original reference=Let%20Awesome.Site%20help%20you%20with%20your%20troubles
```

What would be the best approach to fix this?

**Comment by hollandben** [comment]
This is a good example of the problems currently in the plugin - it searches the given file for references to what it believes are real files. Looks like the plugin is picking up `Awesome.Site` as a file as it matches a regex pattern.

My advice would be to try the `feature/v1.0.0` branch and have a quick read through #147 as it explains the changes that are going to be made. I'm hoping to push it out soon as quite a few people have tested it and the majority of kinks have been worked out.

**Comment by hollandben** [comment]
Fixed in v1.0.0
